codalab/lib/formatting.py

```diff
@@ -37,7 +37,7 @@ def size_str(size, include_bytes=False):
     include_bytes: whether or not to include 'bytes' string in the return value
     Return a human-readable string.
     """
-    if size is None:
+    if size is None or size == 0:
```

Comment:
Why add this?  0 bytes is different from not knowing the size.

Reply:
From a formatting perspective, I feel that showing a size of `'0'` isn't useful information for the user. So instead of always checking for size `'0'` everywhere, I thought it would be better to just not return a size in that case.

Reply:
There is a meaningful difference between `None` and `0`.  You can have a job that outputs a file of `0` size, which is different from that file not existing.